docs/product_upgrades/JIRA_UPGRADE.md

```diff
@@ -0,0 +1,44 @@
+# Jira Rolling Upgrade
+For the sake of example, let's say we have Jira in version `8.13.0` deployed to our Kubernetes cluster that we want to
+upgrade it to `8.13.1` which we'll call the *target version*. You can substitute the target version for the one you
```

Comment:
I would change to:
Let's say we have Jira version `8.13.0` deployed to our Kubernetes cluster, and we want to upgrade it to version `8.13.1`, which we'll call the *target version*.

Reply:
I didn't like that sentence. Thanks; it's much better now.